src/schemathesis/loaders.py

```diff
@@ -17,7 +19,7 @@ def from_path(path: PathLike, method: Optional[Filter] = None, endpoint: Optiona
 
 def from_uri(uri: str, method: Optional[Filter] = None, endpoint: Optional[Filter] = None) -> BaseSchema:
     """Load a remote resource and parse to schema instance."""
-    response = urlopen(uri)
+    response = urlopen(Request(uri, headers={"User-Agent": f"schemathesis/{schemathesis.__version__}"}))
```

Comment:
Can we reuse the same string in both places where ua is used? E.g some global variable (maybe in a separate ‘constants.py’ module)

Reply:
What do using about using requests here? In runner we can even reuse the same session if we make it as a dependency injection here

Reply:
E.g add an argument ‘session=None’ and if it is None, then ‘session = requests.Session()’

Reply:
E.g add an argument ‘session=None’ and if it is None, then ‘session = requests.Session()’

Reply:
urllib can be used for `file://`, while requests can't be used for that. Do we use somehow this kind of URI?

Reply:
Hm, no, it wasn’t documented. But ‘from_file’ loader should be used for it. Also there is an issue for file support in CLI - #119

Reply:
Hm, no, it wasn’t documented. But ‘from_file’ loader should be used for it. Also there is an issue for file support in CLI - #119

Reply:
I’d go for replacing, since it wasn’t documented and there is a proper alternative for this case

Reply:
Ok, on it.